src/payment/checkout/payment-form/CardDetails.jsx

```diff
@@ -209,7 +213,16 @@ export class CardDetailsComponent extends React.Component {
           />
         </h5>
 
-        <div className="row">
+        {this.props.flexMicroformEnabled && <FlexMicroform
```

Comment:
The flag should probably have 3 states - loading, true and false

Reply:
Regarding states - the best practice is generally to have a state/status variable that uses string constants to represent different states.  It turns out it's a bit of an anti-pattern to use booleans for states, because when you find out you have more than two like this, you start creating more booleans, and then the number of combinations becomes far more than the number of valid states your app actually has. :)